make/src/main.rs

```diff
@@ -130,6 +209,11 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             }
         }
 
+        if keep_going {
+            eprintln!("make: Target {} not remade because of errors", target);
+            status_code = 2;
+        }
```

Comment:
From the code, looks like `keep_going` flag doesn't really work, as on an error, the `status_code` is set to 2 and the loop is immediately broken.

Correct me if I'm wrong